v2/tools/asoctl/internal/importing/resource_importer.go

```diff
@@ -44,12 +47,17 @@ func (ri *ResourceImporter) Add(importer ImportableResource) {
 
 // AddARMID adds an ARM ID to the list of resources to import.
 func (ri *ResourceImporter) AddARMID(armID string) {
-	importer := NewImportableARMResource(armID, ri.client, ri.scheme)
+	//TODO: Do we want a way to specify the owner here? Maybe a commandline switch?
```

Comment:
I don't follow this TODO: The owner is implicitly specified via the `armID`, isn't it?

Reply:
It made sense at the time, but I don't remember the details. Removed.